mk/check.mk

```
@@ -89,6 +89,15 @@ check: format lint ## Dev: Run code checks (go fmt, go vet, ...)
 	fi; \
 	if [ "$$FAILED" = true ]; then exit 1; fi
 
+.PHONY: check/rbac
+check/rbac:
+	@RBAC_CHANGED=$$(git --no-pager diff --name-only -- deployments/ | xargs grep -E 'kind: (Role|RoleBinding|ClusterRole|ClusterRoleBinding)' || true); \
+	UPGRADE_CHANGED=$$(git --no-pager diff --quiet UPGRADE.md; [ $$? -ne 0 ] && echo true || echo ""); \
```

Did the previous one work? don't we need `--` before UPGRADE.md?

```suggestion
	UPGRADE_CHANGED=$$(git --no-pager diff --quiet master -- UPGRADE.md; [ $$? -ne 0 ] && echo true || echo ""); \
```